spec/30-processing-model.md

```diff
@@ -0,0 +1,63 @@
+# Processing Model
+
+This section provides a step-by-step exemplary description of the behavior of a tracing
+implementation - also called tracer - when a request is received, processed and 
+potentially forwarded. This description can be used as a reference when
+implementing a Trace Context compliant tracing system, middleware - like a proxy
+or messaging bus - or cloud service.
+
+## Processing Model for Working with Trace Context
+
+The processing model describes the behavior of a tracer which modifies and
+forwards Trace Context headers.
+
+1. The tracer checks an incoming request for a `traceparent` and a `tracestate`
+   header.
+2. If no `traceparent` header is received, the tracer creates a new `trace-id`
+   and `parent-id` representing the current request.
+3. If `traceparent` header is present, the tracer tries to parse the version of
+   the `traceparent` header.
+   - If the version cannot be parsed, the tracer creates a new
+     `traceparent` header and deletes `tracestate`.
+   - If the version number is higher than supported by the tracer, the
+     implementation uses the format defined in this specification to parse
+     `trace-id` and `parent-id`. The tracer will only parse `trace-flags` values
+     supported by the current version of this specification and ignore all other
+     values. If parsing fails, the tracing system creates a new `traceparent`
+     header and deletes the `tracestate`.
+4. If the tracer supports the version number, it validates `trace-id` and
+   `parent-id`.
+   - If either `trace-id`, `parent-id` or `trace-flags` are invalid, the tracer
+     creates a new `traceparent` header and deletes `tracestate`.
+5. The tracer validates the `tracestate` header. If the `tracestate` header
```

Comment:
Should we also add something about validating the size of tracestate?

Reply:
Looks good to me otherwise

Reply:
We decided to leave size handling open and have everyone decide how much they want to forward. So not entirely sure what to add.

Reply:
Makes sense, thanks!